build/js/inverted-index.js

```diff
@@ -14,154 +14,168 @@ class InvertedIndex {
 
 
   /**
-   * converts the string provided to lower case, strips the string of all 
+   * converts the string provided to lower case, strips the string of all
    * special characters, extra spaces and converts the result into an Array
-   * @static 
    * @param {string} text the text to be converted
    * @returns {Array} the result that will be returned after function call
    */
-  static textToArray(text) {
-    const result = text.toLowerCase().replace(/[^a-z0-9\s]/g, "").split(/\s+/);
+  textToArray(text) {
+    const result = text.toLowerCase().replace(/[^a-z0-9\s]/g, '').split(/\s+/);
     return result;
   }
 
-
   /**
-   * loops through an array and removes duplicate elements
-   * @static
-   * @param {}
-   * @returns {Array} 
+   * check the documents in the uploaded file and returns
+   * the content of the file in a tokenized format
+   * @param {Array} file the content of the file
+   * @returns {Array} returns a boolean
```

Comment:
Your returns specifies a type of Array while your descriptions says you're returning a Boolean. 
Rectify accordingly